### ISSUE TITLE
how can I train without a val_set? thanks

### ISSUE THREAD

**Issue by jarvis1890** [timeseriesAI/tsai#671]

**Comment by oguiza** [comment]
Hi @jarvis1890,
Both `fastai` and `tsai`  are built with the idea of using a validation set. 
However, there's an easy hack you can use, and it's to pass [0] as your validation split and disregard the validation metrics. Here's an example of how this could work: 
```python
from tsai.basics import *
X, y, _ = get_UCR_data('LSST', split_data=False)
splits = (np.arange(len(y)), [0])
tfms = [None, TSClassification()]
batch_tfms = TSStandardize(by_sample=True)
dls = get_ts_dls(X, y, splits=splits, tfms=tfms, batch_tfms=batch_tfms)
learn = ts_learner(dls, metrics=accuracy, cbs=[ShowGraph()], train_metrics=True)
learn.fit_one_cycle(10, 1e-2)
```

**Comment by oguiza** [comment]
I've modified the dataloaders so you no longer need to pass a validation or test set. You can just pass any of the following: 

* a single train_split
* a tuple: (train_split, )
* a tuple: (train_split, None)
* a tuple:  (train_split, [])

**Comment by oguiza** [comment]
I'll close this issue due to the lack of response.
